models/gc_wsi_bgseg/meta.json

```diff
@@ -0,0 +1,156 @@
+  "name": "gc_wsi_bgseg",
+  "title": "Tissue-Background segmentation in histopathological whole-slide images",
+  "summary": {
+    "description": "This algorithm segments the background and tissue in histopathological whole-slide images.",
+    "inputs": [
+      {
+        "label": "Whole-slide image",
+        "description": "Whole-slide histopathology image",
+        "format": "DICOM",
+        "modality": "SM",
+        "bodypartexamined": "WHOLEBODY",
+        "slicethickness": "",
+        "non-contrast": false,
+        "contrast": false
+      }
+    ],
+    "outputs": [
+      {
+        "type": "Prediction",
+        "valueType": "Segmentation mask",
+        "label": "WSI background and tissue segmentation mask",
+        "description": "WSI background and tissue segmentation mask (MHA). The labels are: 0-background 1-tissue. The output will have a pixel spacing",
+        "classes": []
+      }
+    ],
+    "model": {
+      "architecture": "Fully convolutional network with 7 layers, ReLU activation functions in the first 6 convolutional layers and softmax in the last one. Max pooling was inserted after each of the first 3 convolutional layers.",
+      "training": "supervised",
+      "cmpapproach": "2D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 70
+      },
+      "evaluation": {
+        "vol_samples": 38
+      },
+      "public": false,
+      "external": false
+    }
+  },
+  "details": {
+    "name": "Tissue-Background Segmentation in Histopathological Whole-Slide Images",
+    "version": "1.0.0",
+    "devteam": "Peter Bándi",
+    "type": "Segmentation",
+    "date": {
+      "weights": "2019-12-13",
+      "code": "2024-05-16",
+      "pub": "2019-12-17"
+    },
+    "cite": "Bándi P, Balkenhol M, van Ginneken B, van der Laak J, Litjens G. 2019. Resolution-agnostic tissue segmentation in whole-slide histopathology images with convolutional neural networks. PeerJ 7:e8242",
+    "license": {
+      "code": "AGPL v3.0",
+      "weights": "AGPL v3.0"
+    },
+    "publications": [
+      {
+        "uri": "https://peerj.com/articles/8242/",
+        "title": "Resolution-agnostic tissue segmentation in whole-slide histopathology images with convolutional neural networks"
+      }
+    ],
+    "github": "https://github.com/DIAGNijmegen/pathology-tissue-background-segmentation-processor",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "This algorithm is a segmentation method of the background on whole-slide histopathology images. The algorithm will segment WSI with various stains and resolutions. It takes pixel spacing as input parameter (default 2.0μm) and will create a segmentation of that resolution. The input WSI needs to contain at least one layer of that resolution, with a 25% tolerance. The algorithm can also be found on Grand Challenge [1] (access with verified account).",
```

Comment:
Can we remove the account requirement?

Reply:
This is sadly not possible, due to overuse of users. But you only need a verified account once to have access to any public algorithm on Grand Challenge.

Reply:
Now published